perllib/FixMyStreet/App/Form/Waste/Request.pm

```diff
@@ -59,19 +59,20 @@ has_field submit => (
 
 sub validate {
     my $self = shift;
-    my $any = 0;
 
-    foreach ($self->all_fields) {
-        $any = 1 if $_->name =~ /^container-/ && ($_->value || $self->saved_data->{$_->name});
-        # Kingston special case for change-size-first-page
-        $any = 1 if $_->name eq 'how_many_exchange' && ($_->value || $self->saved_data->{$_->name});
-        # Bexley special case for household_size first page
-        $any = 1 if $_->name eq 'household_size' && ($_->value || $self->saved_data->{$_->name});
-        # Bexley special case for 'parent' containers (wheelie bins with size options)
-        $any = 1 if $_->name =~ /^parent-/ && ($_->value || $self->saved_data->{$_->name});
+    # Bexley has its own validation method for requests
```

Comment:
I feel there's a better way to do this, but I didn't know how to skip this validate() method without also skipping the one in Form/Waste.pm.

Reply:
In your Bexley specific validate() you could call the 'grandparent' validate function directly rather than using next->method, in order to skip this one, which hardcodes the relationship but we know that it's fixed anyway.

Reply:
Would that just be doing something like 'Waste::Form->validate()'? Or is there some kind of syntax using SUPER?

Reply:
You would want to call it as `...Form::Waste::validate($self)`. If you called it as Form::Waste->validate() then the class would be passed to it, not the instance, and it wouldn't have the data to look up anything.